lib/Bio/EnsEMBL/DataCheck/Checks/CheckGenomicAlignCoverage.pm

```diff
@@ -0,0 +1,148 @@
+=head1 LICENSE
+
+Copyright [2018-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckGenomicAlignCoverage;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::Utils::SqlHelper;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME        => 'CheckGenomicAlignCoverage',
+  DESCRIPTION => 'Coverage for a MLSS matches the coverage recorded in the  mlss_tag table',
+  GROUPS      => ['compara', 'compara_multiple_alignments', 'compara_pairwise_alignments'],
+  DB_TYPES    => ['compara'],
+  TABLES      => ['dnafrag', 'genome_db', 'genomic_align', 'method_link', 'method_link_species_set', 'method_link_species_set_tag', 'species_set_header', 'species_tree_node', 'species_tree_node_tag']
+};
+
+
+sub tests {
+  my ($self) = @_;
+    
+  my $overlap_check;
+  
+  if ($self->dba->dbc->dbname =~ /plants|pan/) {
```

Comment:
I would suggest to use the `get_division()` method that @JAlvarezJarreta introduced recently
```suggestion
  if ($self->dba->get_division() =~ /plants|pan/) {
```